test/images/volume/rbd/bootstrap.sh

```diff
@@ -25,10 +25,10 @@
 
 
 # Create /etc/ceph/ceph.conf
-sh ./ceph.conf.sh `hostname -i`
+sh ./ceph.conf.sh "$(hostname -i)"
```

Comment:
These changes are all the same.
Why did I modify and fix something: 
```
In ./test/images/volume/rbd/bootstrap.sh line 28:
sh ./ceph.conf.sh `hostname -i`
                  ^-----------^ SC2046: Quote this to prevent word splitting.
                  ^-----------^ SC2006: Use $(...) notation instead of legacy backticked `...`.
```
For more information:
  https://www.shellcheck.net/wiki/SC2046 -- Quote this to prevent word splitt...
  https://www.shellcheck.net/wiki/SC2006 -- Use $(...) notation instead of le...